uniffi_bindgen/src/bindings/python/templates/TopLevelFunctionTemplate.py

```diff
@@ -15,13 +15,7 @@ def {{ func.name()|fn_name }}({%- call py::arg_list_decl(func) -%}):
         {%- when None %}
         lambda val: None,
         {% endmatch %}
-        # Error FFI converter
-        {%- match func.throws_type() %}
-        {%- when Some(e) %}
-        {{ e|ffi_converter_name }},
-        {%- when None %}
-        None,
-        {%- endmatch %}
+        {% call py::error_ffi_converter(func) %}
```

Comment:
Does there need to be a similar change for async methods?

Reply:
I don't think so - the test has a top-level `async fn aoops() -> Result<(), Arc<ErrorInterface>>`, but I might have missed something.

Reply:
Ahh, that one is already calling `error_ffi_converter`.  Seems good to me.